tools/WebhookProcessor/github_webhook_processor.php

```diff
@@ -33,9 +33,6 @@
 $enable_live_tracking = true;
 $path_to_script = 'tools/WebhookProcessor/github_webhook_processor.php';
 $tracked_branch = "master";
-$trackPRBalance = true;
-$prBalanceJson = '';
-$startingPRBalance = 30;
```

Comment:
This was something I had just learned about now, right now PRs only start at 0. Though, from my view, it doesnt' look like this worked anyway...?

Reply:
It's being overridden by the sercret file for some reason

Reply:
Incredible